UPGRADE-6.3.md

```diff
@@ -0,0 +1,64 @@
+UPGRADE FROM 6.2 TO 6.3
+=======================
+
+# Essentials
+
+## Changed Signatures
+
+See the Changed Interfaces and Method Signatures section whether changes in signatures affect the project.
+
+# Changed Features
+
+## \TShopArticleReview::LoadFromRowProtected()
+
+- This works now with a white-list. You need to overwrite `\TShopArticleReview::getFieldWhitelistForLoadByRow()` if 
```

Comment:
Suggestion:

"When loading data from user input, `\TShopArticleReview::LoadFromRowProtected()` now uses a whitelist instead of a blacklist to avoid users being able to pass data not intended for change. If your project uses custom fields in the `shop_article_review` table, override `\TShopArticleReview::getFieldWhitelistForLoadByRow()` to add these fields to the whitelist."

Reply:
Ok